### PR TITLE
fix: clear persisted focus on entering configuration

### PR DESCRIPTION
**Pull request by steinsebastian**
CXSPA-8638

### CODE REVIEW

**Review by Uli-Tiger — DISMISSED** [review]
lgtme

File: feature-libs/product-configurator/rulebased/components/form/configurator-form.component.ts [hunk 1]
```diff
@@ -38,6 +43,7 @@ import { ConfiguratorExpertModeService } from '../../core/services/configurator-
 export class ConfiguratorFormComponent implements OnInit, OnDestroy {
   protected subscription = new Subscription();
 
+  protected focusService = inject(KeyboardFocusService);
```
Comment (Larisa-Staroverova) [inline feature-libs/product-configurator/rulebased/components/form/configurator-form.component.ts]: Please, change the name of the instance to `keyboardFocusService`

**Review by Larisa-Staroverova — CHANGES_REQUESTED** [review]
@steinsebastian, could you, please, rename the instance?
Otherwise, it looks really good.

File: feature-libs/product-configurator/rulebased/components/form/configurator-form.component.ts (hunk 1 above)
Comment (steinsebastian) [reply feature-libs/product-configurator/rulebased/components/form/configurator-form.component.ts]: Done.

**Review by Larisa-Staroverova — APPROVED** [review]

**Pull request merged**